src/common/components/cards/instance-details-footer.tsx

```diff
@@ -63,8 +63,12 @@ export const InstanceDetailsFooter = NamedFC<InstanceDetailsFooterProps>('Instan
         );
     };
 
+    const handleKeyDown = (event: React.KeyboardEvent<any>) => {
+        event.stopPropagation();
+    };
+
     return (
-        <div className={foot}>
+        <div className={foot} onKeyDown={handleKeyDown}>
```

Comment:
This is because pressing `Enter` on the dropdown was causing the card to be selected as well.